src/overlays/actors/ovl_En_Sth/z_en_sth.c

```diff
@@ -388,6 +383,9 @@ Gfx* EnSth_AllocColorDList(GraphicsContext* play, u8 envR, u8 envG, u8 envB, u8
 }
 
 void EnSth_Draw(Actor* thisx, PlayState* play) {
+    static Color_RGB8 sTunicColors[6] = {
```

Comment:
not related to your changes of course, but is tunic really the best word here? makes it sound like link lol
arent they just shirts?

Reply:
Wikipedia defines tunic as

> a garment for the torso, usually simple in style, reaching from the shoulders to a length somewhere between the hips and the ankles.

I don't think the little bit below the belt makes it a tunic